app/controllers/doctors_controller.rb

```diff
@@ -0,0 +1,62 @@
+class DoctorsController < ApplicationController
+  def index
+    @doctors = Doctor.all
+  end
+
+  def show
+  end
+
+  def create
+    @doctor = Doctor.new(doctor_params)
+    if @doctor.save
+      respond_to do |format|
+        format.json { render json: @doctor }
+        # format.html
+      end
+    end
+  end
+
+  def update
+  end
+
+  def edit
+    @doctor = Doctor.find(params[:id])
+    redirect_to root_path unless current_company.id == @doctor.company.id
+  end
+
+  def login
+    if current_doctor
+      redirect_to current_doctor
+    elsif current_company || current_user
+      redirect_to root_path
+    end
+  end
+  def create_session
+    @doctor = Doctor.find_by(email: params[:email])
+    if @doctor
+      if @doctor.password == Digest::MD5.hexdigest(params[:password])
+        session[:doctor_id] = @doctor.id
+        flash[:notice]='You successful login'
```

Comment:
Surrounding space missing for operator `=`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.